### DISCUSSION TITLE
Pointer instead of mouse cursor

### DISCUSSION THREAD

**Discussion by willhoh** [rustdesk/rustdesk#3792] [Ideas]
We are forced to use FastViewer software at work. There you can show the people to help by a pointer where they should click. The control over the mouse is not taken away from them. You can ask for this control. Would it be possible to add this feature? Best would also be to show the 2 to 3 clicks (right, left and middle click) differently.

**Comment by chiehw** [comment]
Good idea, I used to have this need too.

**Comment by zaher** [comment]
I need it too, because I don't want to control the remote computer i want to help my coworker and point where he should click, we need another mouse pointer to show on remote computer

**Reply by SkyBeam** [reply]
Either second pointer or taking control of the mouse pointer on the remote location is needed. Programs like VNC always control a single pointer. Even though this leaving problems with local users constantly taking focus or re-gaining control by moving the mouse. Ideally a second (different color?) mouse pointer would be shown indicating thet the remote control client is doing. Indicating clicks would be a nice bonus helping in remote-education sessions.
Overall it gives a much better feeling to the user seeking for help if he can actually see where the supporting person is moving with the mouse and what action he's doing rather than just seeing the results of the click.

It is actually already possible in options to see the remote cursor - but it's working only in one direction. So I can see if the person on the other side is moving the mouse, but he can't see my movements currently.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/573
